source/agora/utils/SCPPrettyPrinter.d

```diff
@@ -448,8 +448,8 @@ unittest
     auto pair = WK.Keys.NODE2;
 
     auto qc = QuorumConfig(2,
-        [PublicKey.fromString("GBFDLGQQDDE2CAYVELVPXUXR572ZT5EOTMGJQBPTIHSLPEOEZYQQCEWN"),
-         PublicKey.fromString("GBYK4I37MZKLL4A2QS7VJCTDIIJK7UXWQWKXKTQ5WZGT2FPCGIVIQCY5")]);
+        [PublicKey.fromString("boa1xrv266cegdthdc87uche9zvj8842shz3sdyvw0qecpgeykyv4ynssf79ndd"),
+         PublicKey.fromString("boa1xzq266cegdthdc87uche9zvj8842shz3sdyvw0qecpgeykyv4ynss3dne57")]);
```

Comment:
Those two values seem awfully close to one another, how did you obtain them ?

Reply:
Only the first byte is different for each public address. The first address has the value, 216 for the first byte, and the second one has 128 for that.

Reply:
Please use the previous key's value so we don't have to update the hash, just the string representation.

Reply:
Ah, OK. I will.